src/plugins/data_source_management/public/types.ts

```diff
@@ -140,6 +140,7 @@ export interface DataSourceAttributes extends SavedObjectAttributes {
   title: string;
   description?: string;
   endpoint?: string;
+  dataSourceVersion?: string;
```

Comment:
There is another DataSourceAttributes interface in data source plugin, should we update there as well?

Reply:
yes but maybe later, that one seems to be leveraged for data source display page, this one is focusing on the saving part, we may need to display the version number on the datasource list page, with some UI updates as well, will need to check further.